src/main/java/no/ssb/subsetsservice/SubsetsController.java

```diff
@@ -62,6 +69,175 @@ public SubsetsController(){
         return getFrom(LDS_SUBSET_API, "/"+id);
     }
 
+    @GetMapping("/v1/subsets/{id}/versions")
+    public ResponseEntity<String> getVersions(@PathVariable("id") String id) {
+        return getFrom(LDS_SUBSET_API, "/"+id+"?timeline");
+    }
+
+    /**
+     * Get a list of versions of a subset that starts with {versions}.
+     * So if {versions} is "1.0", then 1.0.0, 1.0.1, 1.0.2 etc will be returned.
+     * If {versions} is "1.0.1", then a list with a single item (1.0.1) will be returned
+     * @param id
+     * @param version
+     * @return
+     */
+    @GetMapping("/v1/subsets/{id}/versions/{version}")
+    public ResponseEntity<JsonNode> getVersion(@PathVariable("id") String id, @PathVariable("version") String version) {
+        ResponseEntity<String> ldsRE = getFrom(LDS_SUBSET_API, "/"+id+"?timeline");
```

Comment:
FIXME: input sanitizing, input validation (Spring Security)
This request will be authorized to communicate to the internal service (LDS), it should be properly checked.
Just for inspiration: what happens if id contains something to redirect the further authorized communication somewhere else? or executed by the app or by the LDS later?

Reply:
Ok, thanks for the heads up!